pkgs/applications/graphics/xournalpp/default.nix

```
@@ -55,6 +55,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Xournal++ is a handwriting Notetaking software with PDF annotation support";
     homepage    = "https://xournalpp.github.io/";
+    changelog   = "https://github.com/xournalpp/xournalpp/blob/v${version}/CHANGELOG.md";
```

```suggestion
    changelog   = "https://github.com/xournalpp/xournalpp/blob/master/CHANGELOG.md";
```

this way the link is clickable

---

But link is not pinned in such case, for example it can be renamed/removed on master.